docs/README.md

```
@@ -0,0 +1,76 @@
+### Claude Code コマンドとの連携
+
+| コマンド                   | 関連ドキュメント                                           |
+| -------------------------- | ---------------------------------------------------------- |
+| `/setup-new-repo`          | [setup/README.md](./setup/README.md)                       |
+| `/setup-ci`                | [setup/README.md](./setup/README.md)                       |
+| `/config-base-sync-update` | [using-config-base-image.md](./using-config-base-image.md) |
+| `/repo-maintenance`        | [tool-catalog.md](./tool-catalog.md)                       |
+
+## ディレクトリ構造
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)